electron/index.js

```diff
@@ -100,11 +100,17 @@ class CapacitorSplashScreen {
       frame: false,
       show: false,
       transparent: this.splashOptions.transparentWindow,
+      webPreferences: {
+        // Required to load file:// splash screen
+        webSecurity: false  
+      }
     });
 
+    let imagePath = path.join(rootPath,'splash_assets', this.splashOptions.imageFileName);
```

Comment:
Don't merge this. It doesn't work on Windows. You end up with a URL that looks like this:
`file://"C:\git\pinpoint\electron\splash_assets\splash.png"` which is obviously not right. We need a URL combine, not a path.join.

Reply:
It's fixed with my last commit and should work now.